schema-bridge.graphql

```diff
@@ -982,7 +982,7 @@ type PoolHourlySnapshot @entity(immutable: true) {
   rewardTokenEmissionsUSD: [BigDecimal!]
 }
 
-type PoolRouteSnapshot @entity(immutable: true) {
+type PoolRouteSnapshot @entity(immutable: true) @regularPolling {
```

Comment:
@lowjiansheng what do you think here? This is both a daily and hourly snapshot...

Reply:
I think having it as an hourlySnapshot would be good. We'll just poll the data every hour instead of every day.